src/main/java/net/reini/rabbitmq/cdi/MessageConverterException.java

```diff
@@ -0,0 +1,20 @@
+package net.reini.rabbitmq.cdi;
+
+/**
+ * Exception for failures in message conversion.
+ * 
+ * @author André Ignacio 
+ */
+public class MessageConverterException extends RuntimeException {
```

Comment:
In order to use it on a `MessageEncoder` or `MessageDecoder` we should rename it to `MessageEncodingException`